hexa/pipelines/graphql/schema.graphql

```diff
@@ -139,10 +141,31 @@ type UploadPipelineResult {
   errors: [PipelineError!]!
 }
 
+input PipelineLogMessageInput {
+  priority: MessagePriority!
+  message: String!
+}
+
+type PipelineLogMessageResult {
+  success: Boolean!
+  errors: [PipelineError!]!
+}
+
+input PipelineProgressInput {
+  percent: Int!
+}
+
+type PipelineProgressResult {
+  success: Boolean!
+  errors: [PipelineError!]!
+}
+
 extend type Mutation {
   createPipeline(input: CreatePipelineInput): CreatePipelineResult!
   deletePipeline(input: DeletePipelineInput): DeletePipelineResult!
   runPipeline(input: RunPipelineInput): RunPipelineResult!
   pipelineToken(input: PipelineTokenInput): PipelineTokenResult!
```

Comment:
Not related to this PR but should be `generatePipelineToken`

Reply:
Will update that later when permission archi is clearer